istiocon 2022 announcement (#10912)
istio/istio.io: content/en/blog/2022/istiocon-2022/index.md

publishdate: 2022-02-14
attribution: "Istio Steering Committee"
keywords: [IstioCon,Istio,conference]
---

IstioCon 2022, set for April 25-29, will be the second annual conference for Istio, the industry’s [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf). This year’s conference will again be 100% virtual, connecting community members across the globe with Istio’s ecosystem. Visit the [conference website](https://events.istio.io/) for all the information related to the event.

{{< image width="75%"
    link="./istioconlogo.jpg"
    alt="IstioCon logo"
    >}}

IstioCon provides an opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and will feature maintainers from across the Istio ecosystem. This year’s IstioCon features sessions focused on sharing real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will range from introductory to advanced levels, split into four main topic tracks:

* **Getting started & getting involved**

> * **Tools, features & functionality** Observability, traceability, and other things built on top of Istio.
> * **Infrastructure & networking** Specific interest in talks addressing performance, cost, and multi-cloud environments.
> * **Tech evolution & what's next** The evolution of Istio, new standards, new extensions, and how to address problems that are interesting to tackle.

Reviewer: ```suggestion
* **Tools, features & functionality:** Observability, traceability, and other things built on top of Istio.
* **Infrastructure & networking:** How Istio works, with deep-dives into performance, cost, and multi-cloud environments.
* **Tech evolution & what's next:** The evolution of Istio, new standards, new extensions, and how to address problems that are interesting to tackle.
```
